### ISSUE TITLE
How to generate 3D semantic fields?

### ISSUE THREAD

**Issue by YandanYang** [WangYixuan12/gendp#2]
Hi,

This is an excellent work! 
Can you share the code to generate 3D semantic fields for each object?
Thanks.

**Comment by WangYixuan12** [comment]
Hi Yandan, thank you for your interest in our work! You can find the code for generating 3D semantic fields in the Google Colab notebook [here](https://colab.research.google.com/drive/1Yk6uDg2So9A3yWALR7mF5dhl_KKR24eh?usp=sharing). I may also release a script for computing 3D semantic fields recently as well.

**Comment by YandanYang** [comment]
Thank you for your prompt response!
I came across the code in [D3fields](https://github.com/WangYixuan12/d3fields/tree/main).
Is this the same code used in this project?

**Comment by WangYixuan12** [comment]
Yes! Since this work is built upon D3fields, you could also refer to the code in D3fields
